content/en/real_user_monitoring/faq/content_security_policy.md

```diff
@@ -48,6 +48,14 @@ connect-src https://*.browser-intake-ddog-gov.com
 
 {{< /site-region >}}
 
+## Session Replay worker
+
+If you are using Session Replay, make sure to allow `blob` workers by adding the following `worker-src` entry:
```

Comment:
I am an noob but should not it says `blob:` what does the `:` means?

Reply:
I would mention CSP at least.

Reply:
`Blob:` is a type of URL, like `data:`, and `https:`.

Reply:
I will rephrase it, there is no such thing as "blob" workers! I meant workers with a `blob:` URL scheme.

This is litterally the CSP page, and we don't mention CSP on each parts, so I think it's fine.

Reply:
Done

Reply:
> I will rephrase it, there is no such thing as "blob" workers! I meant workers with a blob: URL scheme.

perfect

> This is litterally the CSP page, and we don't mention CSP on each parts, so I think it's fine.

Ah yes indeed